### DISCUSSION TITLE
Start_windows starts to autoload a model

### DISCUSSION THREAD

**Discussion by Rajhin** [oobabooga/text-generation-webui#2733] [Q&A] [answered]
My PC crashed after attempting to load a model with wrong memory settings and now every time I start the webUI it starts to load same model again until it crashes and doesn't actually let me connect to it through 127.0.0.1 so I can't even stop it.

I'm having issues finding list of commands or ways to fix this behavior.

**Comment by viperwasp** [comment, marked as answer]
There are a few ways to potentially fix this. First there are several different ways "Ooba" works depending on what version you have. If you are using "Ooba" with the "start_windows.bat" launcher. I think if you add a second model into your models folder. It will than give you a choice of what model to load when you click "start_windows."  You can load a different model to get access to the models within web-ui where you can fix the settings on the corrupt model that is stuck. 

Alternatively if you update Ooba using "Update_windows" or perhaps the "git pull https://github.com/oobabooga/text-generation-webui.git" command if your running the legacy installer I call it. 

At which point "update_windows" I think always should default to not loading any model. It only loads you into the main system you need to manually load the model from within the Web-ui this allows you to change settings manually. 

If none of this seems to help you can navigate to oobabooga > text-generation-webui > models 
and try editing or deleting the config YAML files. These are were the saved setting per models are kept I think. You can edit with notepad. Make a backup of these files before editing if you wish. Deleting them and running the updater might just replace them I don't know.

**Comment by Rajhin** [comment]
Thanks, that helped.
